routers/api/v1/misc/version.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package misc
+
+import (
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/setting"
+)
+
+// ServerVersion shows the version of the Gitea server
+func ServerVersion(ctx *context.APIContext) {
+	ctx.Write([]byte(setting.AppVer))
```

Comment:
As the entire API is JSON, maybe you should do this instead
```go
type serverVersion struct {
    Version string
}
func ServerVersion(ctx *context.APIContext) {
    ctx.JSON(200, &serverVersion{Version: setting.AppVer})
}
```

_Preferably_ the struct should go in `code.gitea.io/sdk` 🙂 (If you do that just call it `api.Version`)